test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1442,23 +1442,23 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				// Select the last core id
 				higherCoreIds := cores[len(cores)-1]
 				// Get cpu siblings from the selected cores and delete the selected cores  from the map
-				cpusiblings := nodes.GetCpuSiblings(numaCoreSiblings, higherCoreIds)
+				cpusiblings := nodes.GetAndRemoveCpuSiblingsFromMap(numaCoreSiblings, higherCoreIds)
```

Comment:
next time: `Pop` may be a shorter sinonym of `GetAndRemove`. I won't resubmit for this.

Reply:
Thanks and noted